test/fixtures/dialects/snowflake/snowflake_inline_comment.sql

```diff
@@ -0,0 +1,6 @@
+//Snowflake Inline Comment
+-- Classic Inline Comment
+SELECT 1;
+# Classic Inline Comment
+SELECT 1//Snowflake Inline
```

Comment:
Call me picky, but most of those aren't inline. What about this:

```suggestion
# Classic Inline Comment
SELECT 1; -- Classic Inline Comment
SELECT 1; # Classic Inline Comment
SELECT 1; //Snowflake Inline Comment
SELECT 1;-- Classic Inline Comment No Space
SELECT 1;# Classic Inline Comment No Space
SELECT 1//Snowflake Inline Comment No Space
```

Reply:
Gumble grumble grumble. Can do - but I contest they are inline - they're inline, if they're not multi-line IMO. I'll push the changes now

Reply:
Fair comment! Kinda badly named in grammar.

Just count yourself lucky I didn't make you add a `SELECT /* IGNORE THIS PART */ 1;` statement as another "inline" comment type 😁

Cheers for persevering with me though!